src/main/java/org/jabref/gui/customjfx/InputMethodSupport.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2013, Oracle and/or its affiliates. All rights reserved.
```

Comment:
@koppor  This is a direct copy from the openfjx code, no modifications, Is this compatible with our license?
http://hg.openjdk.java.net/openjfx/8/master/rt/file/tip/modules/swing/src/main/java/javafx/embed/swing/InputMethodSupport.java